tests/Common/SpanAssertion.php

```diff
@@ -97,6 +97,9 @@ public static function forOperation($name, $error = false, $onlyCheckExistence =
     public function setError($errorType = null, $errorMessage = null, $exceptionThrown = false)
     {
         $this->hasError = true;
+        if (!is_array($this->exactTags)) {
+            $this->exactTags = [];
+        }
```

Comment:
Can you show me an example of why this is needed?

Reply:
The [default for `$this->exactTags`](https://github.com/DataDog/dd-trace-php/blob/8093450efd81fc32f76e109536c226f6ac0c621a/tests/Common/SpanAssertion.php#L15) is a string. So if you don't use `withExactTags()` before calling `setError()`, then `$this->exactTags` will be a string and will raise a warning in the tests. This is needed since the WordPress tests check for errors but not exact tags on the error span.